src/app/frontend/common/components/labels/labels_controller.js

```diff
@@ -19,6 +19,12 @@
 const alwaysVisibleLabelsNumber = 5;
 
 /**
+ * Regular expression matching URLs.
+ * @type {RegExp}
+ */
+const urlRegexp = new RegExp('(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?', 'i');
```

Comment:
Hmm, with regexp it's possible to determine if annotation value is URL or not. Without it we would make all annotations hrefs.